wiki/Tournaments/GBC/GBC_2023_Spring/zh.md

```diff
@@ -40,7 +40,9 @@ tags:
 | *季后赛第九至十二名* | 30 RMB |
 | *季后赛第十三至十六名* | 20 RMB |
 | *季后赛第十七至二十四名* | 10 RMB |
-| *常规赛全勤奖* | 猫猫定制勋章(Pending), 15 RMB(B组), 25 RMB(A组), 35 RMB(S组), 45 RMB(EX组) |
+| *常规赛全勤奖* | 猫猫定制勋章, 15 RMB(B组), 25 RMB(A组), 35 RMB(S组), 45 RMB(EX组) |
+
+![GBC 2023 Spring banner](img/GBC-2023-Spring-badge-1st.png) ![GBC 2023 Spring KanonBot 1st-place badge](img/badge-kanon-1st.png) ![GBC 2023 Spring banner KanonBot 2nd-place badge](img/badge-kanon-2nd.png) ![GBC 2023 Spring KanonBot 3rd-place badge](img/badge-kanon-3rd.png) ![GBC 2023 Spring KanonBot attendance badge](img/badge-kanon-attended.png)
```

Comment:
the alt text in brackets `[]` should be translated. they appear if the images fail to load and may be used by screen readers

Reply:
finished